app/pages/_app.tsx

```diff
@@ -64,6 +65,10 @@ export default class App extends NextApp<AppProps> {
 
     return (
       <Sentry.ErrorBoundary fallback={ErrorFallback}>
+        <SessionTimeoutHandler
+          pageComponent={Component}
+          modalDisplaySecondsBeforeLogout={60}
```

Comment:
```suggestion
          modalDisplaySecondsBeforeLogout={120}
```

This should be 2min according to our acceptance criteria

Reply:
oops! I had changed it for the demo and somehow committed it